test/passing/doc_comments.mli

```diff
@@ -189,7 +210,7 @@ val x : x
 
 (** for [Global]s *)
 
-(** generic command: ∀xs.{foot}-{post} *)
+(** generic command: ∀xs.[foot]-[post] *)
```

Comment:
We need to avoid rewrites like this, they are semantically meaningful. If the original is ill-formed somehow, better to fail.

Reply:
It is:
```
'{abc}': bad markup.
Suggestion: did you mean '{!abc}' or '[abc]'?
```

Reply:
Ok, thanks. I think it would be better to fail in such cases. Or at least warn, or have an option to choose. There are, I expect, lots of docstrings in the wild that are ill-formed, and it isn't only ocamlformat that will be doing the wrong thing with them. So probably better to give users feedback as early as possible. Hopefully we can get line numbers, etc. for an error message.